<?xml version="1.0"?>
<launch>

  <node name="detector" pkg="pencil_perception_module" type="detector.py" output="screen"> 
    <param name="subscribed_topic_name" value="/camera/image_raw"/>
    <param name="published_topic_name" value="/detection_results"/>

    <param name="model_path" value="please_input_your_path_to_perception_model"/>
    <!-- <param name="model_path" value="/home/huy/transfer_learning_ws/src/pencil_filter_processor/pencil_perception_module/model/2022-01-22-13-45-pretrain-single-gate-corrected"/> -->
  
    <!-- Apply pencil filter on the input images -->
    <param name="apply_pencil_filter" type="bool" value="true"/>

    <param name="epoch" type="int" value="100"/>

    <param name="threshold" type="double" value="0.7"/>


  </node>
  

</launch>
